SPECS/mstflint/mstflint.spec

```diff
@@ -23,15 +25,29 @@ BuildRequires:  libxml2-devel
 BuildRequires:  make
 BuildRequires:  openssl-devel
 BuildRequires:  zlib-devel
+Requires:       kernel
 Requires:       python3
 
+# set package name
+%{!?_name: %global _name mstflint}
+
+%global kver %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}-%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-devel))
```

Comment:
Let's just use the hard-coded version of `kernel-devel` we're building this spec with (see: comments about `BuildRequires`) and reduce the complexity.